src/app/ubs/ubs-admin/components/shared/components/tariff-deactivate-confirmation-pop-up/tariff-deactivate-confirmation-pop-up.component.html

```diff
@@ -32,21 +32,31 @@
     <div class="form-block">
       <div class="ua-part">
         <label class="label">{{ 'ubs-tariffs.region' | translate }}</label>
-        <p class="text">{{ regionName }}</p>
+        <p class="text">{{ getLangValue(regionName, regionEnglishName) }}</p>
       </div>
       <div class="en-part">
-        <label class="label">Region</label>
-        <p class="text">{{ regionEnglishName }}</p>
+        <label class="label">{{ getLangValue(regionLabelEn, regionLabelUa) }}</label>
+        <p class="text">{{ getLangValue(regionEnglishName, regionName) }}</p>
       </div>
     </div>
     <div class="form-block">
       <div class="ua-part">
         <label class="label">{{ 'ubs-tariffs.city' | translate }}</label>
-        <p *ngFor="let locationName of locationNames" class="text">{{ locationName }}</p>
+        <div *ngIf="lang">
+          <p *ngFor="let locationName of locationNames" class="text">{{ locationName }}</p>
```

Comment:
You can add another function:
getLangArrayValue(uaValue: string[], enValue[]) {
return this.languageService.getLangValue(uaValue, enValue) as string[];
}
replace locationNames with getLangArrayValue(locationNames, locationEnglishNames)

Reply:
Thank you! Change code according to recommendations